virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -71,6 +71,9 @@ The SVVP Certification applies to:
 
 //CNV-28096 Release note: NEW feature -- clone from cached snapshot
 
+//CNV-28726 Release note: New 
+* By default, Windows 11 virtual machines (VMs) are incompatible with the VM snapshot feature. You can, however, enable snapshots on a Windows 11 VM by configuring its trusted platform module (TPM) device's state as non-persistent. Note that a non-persistent TPM will, in turn, render the VM incompatible with software that relies on TPM persistence, such as BitLocker.
```

Comment:
1. Add the Windows Server 2022 as it has same feature limitations

2. Not sure about the "By default" part here..
Right now both common templates and instance types have non-persistent TPM in spec
So maybe it'll be more accurate to say:
`By default, Windows 11 and Windows Server 2022 have trusted platform module (TPM) device's state as non-persistent. Changing it to persistent, enables software that relies on TPM persistence, such as BitLocker. However, in turn, this will disable VM snapshot feature, as it's currently incompatible with TPM in persistent state`